lang/test/org/partiql/lang/eval/EvaluatorTestBase.kt

```diff
@@ -50,7 +53,6 @@ import kotlin.test.assertEquals
  *
  * As we parameterize PartiQL's other tests, we should migrate them away from using this base class as well.
  */
-@Deprecated("This class and everything in it should be considered deprecated.")
```

Comment:
Always wondered about why `EvaluatorTestBase` was deprecated. Any insight into its history?

Reply:
The intention was to migrate to the new test DSL that I made. That migration however never happened and the deprecation was just causing a ton of compiler warnings.